project/AppDependencies.scala

```diff
@@ -13,14 +13,15 @@ object AppDependencies {
     "io.circe"    %% "circe-generic"                    % circeVersion,
     "io.circe"    %% "circe-parser"                     % circeVersion,
     "io.circe"    %% "circe-yaml"                       % circeVersion,
-    "io.circe"    %% "circe-generic-extras"             % circeVersion
+    "io.circe"    %% "circe-generic-extras"             % circeVersion,
+    "uk.gov.hmrc" %% "ui-test-runner"                   % "0.17.0"
   )
 
   val test = Seq(
     "uk.gov.hmrc"       %% s"bootstrap-test-$playVersion" % bootstrapVersion % "test",
-    "org.scalatestplus" %% "selenium-4-2"                 % "3.2.13.0"       % "test",
+    "org.scalatestplus" %% "selenium-4-12"                % "3.2.17.0"       % "test",
     "org.mockito"       %% "mockito-scala-scalatest"      % "1.17.30"        % "test",
     "org.jsoup"          % "jsoup"                        % "1.10.2"         % "test",
-    "uk.gov.hmrc"       %% "webdriver-factory"            % "0.41.0"         % "test"
+    "uk.gov.hmrc"       %% "webdriver-factory"            % "0.46.0"         % "test"
```

Comment:
@kyle-bowden we should be able to remove webdriver-factory now - ui-test-runner replaces it